src/Sylius/Bundle/CoreBundle/Resources/config/app/fixtures.yml

```diff
@@ -9,119 +9,116 @@ sylius_fixtures:
+                                enabled: true
 
                 shipping_method:
                     options:
-                        random: 20
+                        custom:
+                            ups:
+                                name: UPS
+                                enabled: true
+                            dhl_express:
+                                name: DHL Express
+                            fedex:
+                                name: FedEx
 
                 channel:
                     options:
-                        random: 3
+                        custom:
+                            us_web_store:
+                                name: US Web Store
+                                locales:
+                                    - "%locale%"
+                                currencies:
+                                    - "%currency%"
+                                shipping_methods:
+                                    - ups
+                                    - dhl_express
+                                    - fedex
+                                payment_methods:
+                                    - cash_on_delivery
+                                    - bank_transfer
 
                 regular_user:
                     name: user
                     options:
                         random: 20
 
+                rbac: ~
                 admin_user:
                     name: user
                     options:
                         custom:
                             -
                                 email: sylius@example.com
-                                first_name: Administrator
-                                last_name: Sylius
+                                first_name: John
+                                last_name: Doe
                                 password: sylius
                                 enabled: true
                                 admin: true
 
-                product_option:
-                    options:
-                        random: 10
-
-                product_attribute:
-                    options:
-                        random: 10
-
-                product_archetype:
-                    options:
-                        random: 10
-
                 tax_category:
                     options:
-                        random: 10
+                        custom:
+                            clothing:
+                                name: Clothing
+                            books:
+                                name: Books
+                            other:
+                                name: Other
 
                 tax_rate:
                     options:
-                        random: 10
-
-                taxon:
-                    options:
-                        random: 50
+                        custom:
+                            clothing_tax:
+                                name: Clothing Sales Tax 7%
```

Comment:
It will be strange to have tax rate named `Clothing Sales Tax 7%` with random rate from 0 to 40%. Maybe simply remove `7%` from name?